docs/reference/modules/attachment.md

```diff
@@ -10,75 +10,120 @@ extends: '@apostrophecms/module'
 |  Property | Type | Description |
 |---|---|---|
-| [`copyOriginal` *](#copyoriginal) | Boolean | Undefined by default. If set to `false` the original image will not be copied to the database, only scaled images. |
-| [`fileGroups`](#filegroups) | Array | Assigns uploaded files to either an 'image' or 'office' category to determine post-upload manipulation. |
-| [`image` *](#image) | String \|\| Object | Sets the image processor to 'sharp' (by default), 'imagemagick, or a custom processor passed in an object. |
-| [`imageSizes`](#imagesizes) | Object | Takes an object with `add` and `remove` properties to change the image sizes created upon image import. |
-| `orientOriginals` * | Boolean | Unless set to `false`, the uploaded image will be reoriented according to header data. |
-| [`postprocessors` *](#postprocessors) | Array | Takes an array of objects detailing optional postprocessors for images. |
-| `scaledJpegQuality` * | Integer | Sets the JPEG quality setting for scaled images - defaults to 80. |
-| `sizeAvailableInArchive` * | String | Takes the name of a size to make available even if file access is disabled. Defaults to 'one-sixth'. |  
-
-### `copyOriginal`
+| [`fileGroups`](#filegroups) | Array | Assigns uploaded files to either an 'image' or 'office' category to allow for upload and manipulation. |
 
-This option should be passed to the `@apostrophecms/uploadfs` module. By default, the `copyOriginal` value is undefined, resulting in storage of the original image in the database. Explicitly setting this to `false` will block this behavior.
+### `fileGroups`
 
-#### Example
+This option should be passed to the `@apostrophecms/attachment` module. By default, this option is set to an array with two objects and any array passed through this option will replace the default values.
 
 <AposCodeBlock>
 
 ```javascript
-module.exports = {
-  options: {
-    uploadfs: {
-      copyOriginal: false'
-    }
-  }
-};
+...
+    self.fileGroups = self.options.fileGroups || [
+      {
+        name: 'images',
+        label: 'apostrophe:images',
+        extensions: [
+          'gif',
+          'jpg',
+          'png',
+          'svg',
+          'webp'
+        ],
+        extensionMaps: { jpeg: 'jpg' }
+      },
+      {
+        name: 'office',
+        label: 'apostrophe:office',
+        extensions: [
+          'txt',
+          'rtf',
+          'pdf',
+          'xls',
+          'ppt',
+          'doc',
+          'pptx',
+          'sldx',
+          'ppsx',
+          'potx',
+          'xlsx',
+          'xltx',
+          'csv',
+          'docx',
+          'dotx'
+        ],
+        extensionMaps: {}
+      }
+    ];
+...
 ```
 <template v-slot:caption>
-  module/@apostrophecms/uploadfs/index.js
+  /@apostrophecms/attachment/index.js
```

Comment:
name is right now, but the path is not (module/whatever/index.js is still the right thing to suggest)

Reply:
Changed.